rpc_jobs/standard_job_premerge.yml

```diff
@@ -26,6 +26,7 @@
     FLAVOR: "performance1-1"
     IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
     TRIGGER_PR_PHRASE_ONLY: false
+    skip_list: []
```

Comment:
What if we changed the default to ...

```
    skip_list:
      - '.*md\$'
      - '.*rst\$'
      - '^releasenotes/'
      - '^gating/generate_release_notes/'
      - '^gating/post_merge/'
      - '^gating/update_dependencies/'
```

... and then allowed projects to override if necessary?

Reply:
+1

Reply:
I think we need to be careful here. We cannot know for sure that test code in `gating/post_merge_test/` is not referenced by some other testing. In the case of rpc-openstack `gating/post_merge_test/` is a symlink and so this is fine but that might not be how all projects structure this code and it would be better to skip too little than too much.

Reply:
Yes, I specifically only implemented this skip list for rpc-openstack because that's what always was there. I don't think we can assume that other projects want to skip the same things. Also bear in mind that most projects only have one premerge project defined, so it's not much to manage. RPC-O has many because it's maintaining so many branches. Eventually that will change, so having those repeated bits is a small price.